spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -606,12 +608,18 @@ def scale_image(self):
             height = (size.height() -
                       style.pixelMetric(QStyle.PM_LayoutTopMargin) -
                       style.pixelMetric(QStyle.PM_LayoutBottomMargin))
-            if (fwidth / fheight) > (width / height):
-                new_width = int(width)
-                new_height = int(width / fwidth * fheight)
-            else:
-                new_height = int(height)
-                new_width = int(height / fheight * fwidth)
+            try:
+                if (fwidth / fheight) > (width / height):
+                    new_width = int(width)
+                    new_height = int(width / fwidth * fheight)
+                else:
+                    new_height = int(height)
+                    new_width = int(height / fheight * fwidth)
+            except ZeroDivisionError:
+                icon = ima.icon('broken_variant')
+                self.figcanvas._qpix_orig = icon.pixmap(fwidth, fheight)
```

Comment:
Let's use a fixed sized here. Perhaps `icon.pixmap(200, 100)`?

@steff456, please try different combinations of width and height to have a reasonable (but not so big) image size.

Reply:
Also, could you add a tooltip to this icon that says something like `The image is broken, please try to regenerate it`?

It'd be nice to have something like that to let users know why we're showing this image instead of the one they expected to see.

Reply:
If I change the size, the icon gets ugly. It is like if I fix the size then it will cover the whole canvas and I cannot resize the QFrame.
![image](https://user-images.githubusercontent.com/20992645/80414138-2e363d80-8896-11ea-809f-2567a1df5e5e.png)

Reply:
Ok, I see. That's because the image is rescaled to fit inside the pane. Then your previous solution is fine.

What about the tooltip?